server/src/models/schema.sql

```diff
@@ -0,0 +1,16 @@
+
+CREATE TYPE roleType AS ENUM ('SuperAdmin', 'Admin', 'Subscriber') DEFAULT 'Subscriber';
+
+CREATE TABLE users(
+    user_id SERIAL PRIMARY KEY,
```

Comment:
study benefits of uuid for unique ID generation

Reply:
study and implement of uuid on database